docs/content/Getting-Started/Migrate-from-Core/Import-GitHub-repository.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: 'Getting Started with Cube Cloud: Import GitHub repository'
+title: 'Import a GitHub repository to Cube Cloud'
```

Comment:
@hassankhan Let's edit all "import" guides' titles this way. Reads better, right?